commit ec869453ecbb4a71611392b077b6c3f5fc093712
Author: Balló György <ballogyor@gmail.com>
Date:   Tue Oct 7 21:28:55 2014 +0200

    lxsession-logout: Fix user switch with LightDM when GDM is installed.
    
    We can't rely on gdmflexiserver, because GDM's version is incompatible with LightDM.

diff --git a/lxsession-logout/lxsession-logout.c b/lxsession-logout/lxsession-logout.c
index 9ace78d..a928118 100644
--- a/lxsession-logout/lxsession-logout.c
+++ b/lxsession-logout/lxsession-logout.c
@@ -530,21 +530,7 @@ int main(int argc, char * argv[])
         handler_context.switch_user_GDM = TRUE;
     }
 
-    /* lightdm also use gdmflexiserver */
-    if (verify_running("lightdm", "gdmflexiserver"))
-    {
-        handler_context.switch_user_available = TRUE;
-        handler_context.switch_user_GDM = TRUE;
-    }
-
-    /* lightdm also use gdmflexiserver */
-    if (verify_running("lightdm", "gdmflexiserver"))
-    {
-        handler_context.switch_user_available = TRUE;
-        handler_context.switch_user_GDM = TRUE;
-    }
-
-    /* lightdm can also be find by the env */
+    /* lightdm can be find by the env */
     if (g_getenv("XDG_SEAT_PATH"))
     {
         handler_context.switch_user_available = TRUE;
